src/jsifier.js

```diff
@@ -45,7 +47,8 @@ function JSify(data, functionsOnly) {
     // else. This lets us not hold any strings in memory, we simply print
     // things out as they are ready.
 
-    var shellParts = read(shellFile).split('{{BODY}}');
+    var thirdParty = thirdPartyFiles.map(function(f) { return read(f) }).join('\n');
+    var shellParts = read(shellFile).replace('{{THIRD_PARTY}}', thirdParty).split('{{BODY}}');
```

Comment:
I agree with @juj, we should avoid adding a dependency here if we can. It should be easy enough to write code for this, the code you quoted looks like it could be a basis. For testing, I think there is a test for MEM_INIT_METHOD (can search under tests/) which checks a large variety of patterns, which we can maybe repurpose. If not, we should something like that.